crates/e2e/src/xts.rs

```diff
@@ -56,7 +57,7 @@ use subxt::{
     },
     tx::{
         Signer,
-        SubmittableExtrinsic,
+        SubmittableTransaction,
```

Comment:
Renamed in https://github.com/paritytech/subxt/commit/b6b9ac65c76f2ad8246b4f55283d81be2c2ec000